### PR TITLE
Feature/responsiveness

### PR DESCRIPTION
**Pull request by sarveshpro**
Fixes #

Changes in this pull request:
-

### CODE REVIEW

**Comment by sagararyal** [comment]
Screen Shot 2020-09-13 at 9 48 05 PM


the transform events takes too much memory. We don't need transform for individual divs

Also the wobbling is still there for scroll

**Comment by sarveshpro** [comment]
Will change the scroll implementation. this won't work properly.

File: pages/index.tsx [hunk 1]
```diff
@@ -52,8 +52,6 @@ export default function Donate() {
     return () => window.removeEventListener('wheel', handleScroll);
   });
   return (
-    <Layout>
-      {projects ? <ProjectsList {...DonateProps} /> : <h2>Loading...</h2>}
-    </Layout>
+    <Layout>{projects ? <ProjectsList {...DonateProps} /> : <h2></h2>}</Layout>
```
Comment (Plant-for-the-Planet) [inline pages/index.tsx:55]: can we make the screen color same as map blue when this loads?
or instead load the project loader here?

Comment (sarveshpro) [reply pages/index.tsx:55]: I am sure this is done in fix scroll PR. the color is not accurate though I have messaged Killian to get the actual color.

Comment (sarveshpro) [reply pages/index.tsx:55]: also, scroll PR should have all the changes in this PR so instead merge that and close this PR.

**Pull request merged**